lib/tenstorrent/bh_arc/regulator_config.c

```diff
@@ -272,6 +272,7 @@ static const RegulatorData serdes_vr_data[] = {
 
 /***** End of common serdes VR settings */
 
+/* p100 configuration removed - p100 no longer supported
```

Comment:
We should just remove things instead of commenting them. They are still preserved in git history.